backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokensResponse.java

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package com.google.gpay.virtualqueue.backendservice.proto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import com.google.gpay.virtualqueue.backendservice.model.Token;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class GetTokensResponse {
+    private UUID shopId;
+    private List<Token> tokenList = new ArrayList<>();
```

Comment:
We should not have initializations in the proto classes.

Reply:
Removed it!